src/groups/mqb/mqbblp/mqbblp_remotequeue.cpp

```diff
@@ -1176,12 +1180,17 @@ void RemoteQueue::onAckMessageDispatched(const mqbi::DispatcherAckEvent& event)
     if (d_state_p->isAtMostOnce()) {
         // Consider this a "non-soft" ACK for all previous broadcasted PUTs.
         size_t numErased = 1 + erasePendingMessages(it);
+        d_expiredPutNum += numErased;
 
         erasePendingMessage(it);
 
-        BALL_LOG_INFO << d_state_p->uri() << ": erased window of " << numErased
-                      << " cached broadcasted PUTs upon "
-                      << bmqt::AckResult::toAscii(ackResult);
+        if (d_throttledCachedPutMessages.requestPermission()) {
+            BALL_LOG_INFO << d_state_p->uri() << ": erased window of "
```

Comment:
This could very well be `BALL_LOG_DEBUG`.  This log does not have much importance.

Reply:
Will redo it to debug then and remove the counter. Let's keep it simple.